client/src/components/Login.js

```diff
@@ -10,7 +10,7 @@ class Login extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      isAuthSuccessful: false,
+      authSucess: false,
```

Comment:
success :P

Reply:
omg my spelling
english is such a struggle 😢